python/metashape_workflow_functions.py

```diff
@@ -1062,18 +1059,24 @@ def build_dem_orthomosaic(self):
                         source_data=Metashape.ElevationData,
                         image_compression=compression,
                     )
-                    if (
-                        self.cfg["buildOrthomosaic"]["enabled"]
-                        and "DSM-mesh" in self.cfg["buildOrthomosaic"]["surface"]
-                    ):
-                        self.build_export_orthomosaic(file_ending="dsm-mesh")
 
-        # Building an orthomosaic from the mesh does not require a DEM, so this is done separately, independent of any DEM building
-        if (
-            self.cfg["buildOrthomosaic"]["enabled"]
-            and "Mesh" in self.cfg["buildOrthomosaic"]["surface"]
-        ):
-            self.build_export_orthomosaic(from_mesh=True, file_ending="mesh")
+        if self.cfg["buildOrthomosaic"]["enabled"]:
+            for surface in self.cfg["buildOrthomosaic"]["surface"]:
```

Comment:
Comments in the next few lines would be helpful, especially if someone has little to no understanding of the inner workings photogrammtery.